### DISCUSSION TITLE
Self hosted new release announcement channel?

### DISCUSSION THREAD

**Discussion by rphlo** [plausible/analytics#2065] [Self-Hosted Support] [answered]
Hi,
First thanks for your project.
Second I'd like to know what is the best way to get informed about the release of new versions of the self hosted package?Newsletter, twitter, here?
I understand the release is happening twice a year and soon should be a new version I am excited about, so I'd like to jump in as soon I can...

**Comment by metmarkosaric** [comment, marked as answer]
hi @rphlo! new releases are announced here so you can watch it: https://github.com/plausible/analytics/releases

**Reply by lightson** [reply]
Silly question perhaps; but latest is showing as 20 Jan 2022. Have there really been no changes in the last seven months?

**Reply by metmarkosaric** [reply]
that's correct. our self-hosted version is not updated as frequently as the Plausible managed hosting in the cloud, which changes daily. this means that the latest features won't be immediately available. think of self-hosted as a long-term release. we stack all the new features and fixes that have been battle-tested on the hosted version and release a new self-hosted version twice a year or so. we don't have any more specific timelines than this as we have limited resources, and the self-hosted release depends on the free capacity of our developers

**Reply by lightson** [reply]
That makes sense; and completely understandable since I guess self-hosted is a pretty direct threat to revenue. I'm quite impressed it exists as a free option to be honest. For my use with a handful of smaller hobby sites it certainly ticks the right boxes and seems extremely similar to the hosted version I tried first.

**Comment by mannp** [comment]
Appreciate the reasoning behind the long term release schedule, but presumably that means the bugs / basic library updates would remain for these extended periods of time?
